shopinvader_api_quotation/routers/cart.py

```diff
@@ -0,0 +1,26 @@
+from typing import Annotated
+
+from fastapi import APIRouter, Depends
+
+from odoo import api
+
+from odoo.addons.base.models.res_partner import Partner as ResPartner
+from odoo.addons.fastapi.dependencies import (
+    authenticated_partner,
+    authenticated_partner_env,
+)
+from odoo.addons.shopinvader_schema_sale.schemas.sale import Sale
+
+cart_router = APIRouter(tags=["carts"])
```

Comment:
Do not redifine a new cart router
Use the existing one
```suggestion
from odoo.addons.shopinvader_api_cart.routers import cart_router
```
 from odoo.addons.shopinvader_api_cart.routers import cart_router